src/logger.ts

```diff
@@ -31,10 +31,11 @@ function printf(timestamp = false) {
 // Setup a default console logger.
 export const logger = winston.createLogger({
   level: "debug",
+  exitOnError: false,
   transports: [
     new winston.transports.Console({
       debugStdout: true,
-      level: "debug",
+      level: "error",
```

Comment:
Yea so my thought is that maybe something above "debug" is getting rejected and causing the connection to timeout. Are you thinking maybe adding a new transport in the flourish app instead?

Reply:
I think if we do this, only error messages would show up (should be easy to test running API locally). I do think the exit one above might be the right solution, or awaiting all log statements (🤮)

Reply:
Yea I agree with not awaiting all log statements. I guess I'm misunderstanding though. Is the concern that number of error messages > the number of debug statements? Or that this will only trigger for "error" levels?

Reply:
The latter, that we won't get console logs for debug/info/warn if we set that to error. And I don't think the console logging is where this error comes from. I was guessing it'd be in @google-cloud/logging-winston (in server.ts). Like this maybe? https://github.com/googleapis/nodejs-logging-winston/issues/751. Also maybe worth updating that package like they suggest.

Reply:
Ah, so I think that the level cascades down. It's listed here: https://github.com/winstonjs/winston#using-logging-levels

And this is the example they use:
`const logger = winston.createLogger({
  levels: winston.config.syslog.levels,
  transports: [
    new winston.transports.Console({ level: 'error' }),
    new winston.transports.File({
      filename: 'combined.log',
      level: 'info'
    })
  ]
});`

That being said, I'll try the suggestion in that post first and upgrade winston. We can monitor the logs over the next few days to see if the errors continue to trigger.

Reply:
```everything info and below to a file (which includes error messages)```

So "info" includes "error", "error" does not include "info"

Reply:
Ah, I seem to have it backwards. I can close this since we're handling it differently for now.